kernel/kernel-defaults/src/test/java/io/delta/kernel/defaults/utils/DefaultKernelTestUtils.java

```diff
@@ -60,10 +59,46 @@ public static Object getValueAsObject(Row row, int columnOrdinal) {
             return row.getBinary(columnOrdinal);
         } else if (dataType instanceof StructType) {
             return row.getStruct(columnOrdinal);
-        } else if (dataType instanceof MapType) {
-            return row.getMap(columnOrdinal);
-        } else if (dataType instanceof ArrayType) {
-            return row.getArray(columnOrdinal);
+        }
+
+        throw new UnsupportedOperationException(dataType + " is not supported yet");
+    }
+
+    /**
+     * Get the value at given {@code rowId} from the column vector. The type of the value object
+     * depends on the data type of the {@code vector}.
+     */
+    public static Object getValueAsObject(ColumnVector vector, int rowId) {
```

Comment:
Moved over from kernel/kernel-defaults/src/main/java/io/delta/kernel/defaults/internal/data/vector/VectorUtils.java since it was only being used in test code after these changes